src/Components/CommentsList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useParams } from "react-router-dom";
+import myApi from "./api";
+import { useState } from "react";
+
+export default function CommentsList() {
+  const { article_id } = useParams();
+  const [commentsList, setCommentsList] = useState([]);
+
+  myApi
```

Comment:
Was the intention to have this occur on render? If so I think a useEffect would be useful here, currently React doesn't have control over the timing of this api call.

It would also be good to extract all of our api calls out into their own file api.js in the root so we can keep our components tidy and allow them to be reusable and able to be configured in isolation.